### PR TITLE
Load topics from .mdb dump

### PR DESCRIPTION
**Pull request by catherinedevlin**
Includes a new `python manage.py importtopics` management command

### CODE REVIEW

File: sbirez/management/commands/importtopics.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    """
+
+    def add_arguments(self, parser):
+
+        parser.add_argument('mdb_filename')
+        parser.add_argument('solicitation_name')
+        # example: DoD SBIR 2015.1
+        parser.add_argument('--clear', action='store_true',
+                            default=False,
+                            help='Firts, delete all records with this solicitation')
```
Comment (DavidEBest) [inline sbirez/management/commands/importtopics.py]: typo: 'First'

File: sbirez/management/commands/importtopics.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+    def handle(self, mdb_filename, solicitation_name, clear=False, **options):
+
+        if not os.path.isfile(mdb_filename):
+            raise OSError("MS Access database %s not found" % mdb_filename)
+
+        # Confirm that solicitation_name is for solicitation in the database
+        proc = subprocess.Popen(#'ls',
+                                '/usr/bin/psql -t -c "SELECT name FROM sbirez_solicitation" %s' %
```
Comment (DavidEBest) [inline sbirez/management/commands/importtopics.py]: Is there a unix-y way of getting the path for psql? I've got it installed in `/usr/local/bin/psql`.

**Comment by DavidEBest** [comment]
Is this still WIP?

**Comment by catherinedevlin** [comment]
This is finally ready for review!  Thanks!

**Comment by DavidEBest** [comment]
It looks like this still has the hard coded psql binary. Is there a better way to reference psql? (See my previous comment)

**Comment by catherinedevlin** [comment]
It looks like using `shell=True` in the arguments to `Popen` lets subprocess see your PATH, so specifying the full path to `pdb` becomes unnecessary.  This should work now.

**Pull request merged**